src/service.js

```diff
@@ -114,6 +118,16 @@ export function installServiceInEmbedScope(embedWin, id, service) {
   );
   registerServiceInternal(embedWin, embedWin, id, () => service);
   getServiceInternal(embedWin, id); // Force service to build.
+  const ampdocFieExperimentOn = isExperimentOn(topWin, 'ampdoc-fie');
```

Comment:
Should we put the above code in an `else` block?

Reply:
I figured this way it'd be easier to cleanup the experiment since we won't need to remove the `else` block later.

Reply:
Might be good to avoid unintended side effects from registering the service on the window though.

Reply:
Oh... I see. Misunderstood. Will send a separate pull request for this specifically.